recipes/htslib/develop/meta.yaml

```diff
@@ -0,0 +1,120 @@
+{% set version = "1.9+353_gefa7aa6" %}
+{% set htslib_rev = "efa7aa632f5f3b73362c3f94be79ca5d49d2bfe4" %}
+
+package:
+  name: htslib-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://github.com/samtools/htslib
+  license: MIT
+  summary: C library for high-throughput sequencing data formats.
+
+build:
+  number: 3
```

Comment:
Build number 0?

Reply:
I should have reset when first creating a package with this version - but now people may be testing with the test binary packages I'd rather not drop it to 0 until the version string changes again.